tests/dbshell/test_mysql.py

```diff
@@ -106,9 +153,24 @@ def test_parameters(self):
                 },
                 ['--help'],
             ),
+            (expected_args, None),
         )
 
-    def get_command_line_arguments(self, connection_settings, parameters=None):
-        if parameters is None:
-            parameters = []
-        return DatabaseClient.settings_to_cmd_args(connection_settings, parameters)
+    def test_crash_password_omission(self):
+        # Ensure the exceptions that results from a client crash doesn't expose
+        # the password.
```

Comment:
Nitpick: pluralisation here feels off.

I think it should be:

```suggestion
        # Ensure the exception that results from a client crash doesn't expose
        # the password.
```